tox.ini

```diff
@@ -7,11 +7,14 @@ setenv =
     LDFLAGS=-L{toxinidir}/tmp-build/lib
     C_INCLUDE_PATH={toxinidir}/tmp-build/include
     LD_LIBRARY_PATH={toxinidir}/tmp-build/lib
+passenv =
+    #http://tox.readthedocs.io/en/latest/config.html#confval-passenv=SPACE-SEPARATED-GLOBNAMES
+    *
 commands =
     pip install -v .
     py.test -v --timeout 20 --ignore=tmp-build --import-mode append {posargs}
     # See tests/README.md for additional notes on testing
-    #python examples/integration_test.py <bootstrap-servers> confluent-kafka-testing [<schema-registry-url>]
+    #python examples/integration_test.py --avro-https --conf tests/testconf.json
```

Comment:
Is --avro-https meant as an example here?
It might be better to remove that so that the larger test suite is run by default if people just uncomment the line.